chromadb/test/property/test_embeddings.py

```diff
@@ -433,14 +479,21 @@ def update_embeddings(self, record_set: strategies.RecordSet) -> None:
     )
     def upsert_embeddings(self, record_set: strategies.RecordSet) -> None:
         super().upsert_embeddings(record_set)
+
+        if record_set["ids"] is None:
+            raise ValueError("IDs should not be empty")
+
+        ids = invariants.wrap(record_set["ids"])
```

Comment:
Should we use the record set count method?

Reply:
we use rely on the number of ids to get the number of items within update and upsert